docs/source/best-practices/homequota.rst

```diff
@@ -1,30 +1,35 @@
 *******************************************************
-Staying Under /home/<username> Quota
+Stay Under your Quota at /home/
 *******************************************************
 It is vital to stay under your /home/<username> quota to prevent issues in using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your /home/<username> directory.
 
 Utlilize /scratch and /work Directories
 =======================================================
-For long term research work storage, /work should be used. If your PI does not have space in /work setup, they can request it using the following form https://bit.ly/NURC-NewStorage and can request additional storage using https://bit.ly/NURC-StorageExtension. More details of /work can be found at :ref:`discovery_storage` For temporary job files you can utilize /scratch/<username>. Please be mindful of the purge policy of /scratch which is covered in :ref:`discovery_storage.rst`.
+For long term research work storage, /work should be used. If your PI does not have space in /work setup, they can request it using the following form https://bit.ly/NURC-NewStorage and can request additional storage using https://bit.ly/NURC-StorageExtension. More details of /work can be found at :ref:`discovery_storage`. For temporary job files you can utilize /scratch/<username>. Please be mindful of the purge policy of /scratch which is covered in :ref:`discovery_storage.rst`.
 
 Determine Large Files/Directories in /home/<username>
 =======================================================
 From a compute node, ``srun --pty /bin/bash``, run the following command from your /home/<username> directory ::
     
- du -h --max-depth=1 *
+ du -shc .[!.]* *
```

Comment:
i think ^ and not ! here (but this is an ugly regex to make "for real". not sure i know how as "..foo" is a legit name.

Reply:
I updated the documentation to use ^. Thank you!